src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2919,6 +2914,23 @@ private BoundExpression BindDelegateCreationExpression(ObjectCreationExpressionS
                     return new BoundDelegateCreationExpression(node, boundLambda, methodOpt: null, isExtensionMethod: false, type: type, hasErrors: !conversion.IsImplicit);
                 }
 
+                else if (analyzedArguments.HasErrors)
```

Comment:
It would make a different. Consider the following examples:
```C#
var f = new Action(bogus);
```
and
```C#
Action w = null;
var f = new Action<string>(w = Console.Wr);
```
For both examples `analyzedArguments.HasErrors == true`, for the first one `(object)argument.Type == null`, for the second one, `argument.Type.TypeKind == TypeKind.Delegate`.

If this is removed, the first one will report another `ERR_MethodNameExpected`, which seems worse because `bogus` could be another delegate. However the second one seems better because it also report conflict between `Action` and `Action<string>`

I think that's out of the scope of the issue I intended to fix.